### ISSUE TITLE
Can't start the shoppingcart

### ISSUE THREAD

**Issue by gserb** [krakenjs/kraken-example-with-shoppingcart#20]
Hello,

I'm using OSX with node v6.1.0. 
After sucesfully run `npm install`, `npm start` is failling with  :

```
Running "jshint:files" (jshint) task
Linting controllers/index.js ...ERROR
[L85:C2] W033: Missing semicolon.
} 
Linting controllers/index.js ...ERROR
[L260:C76] W033: Missing semicolon.
        model.cart.continueMessage = res.bundle.get('keepShopping') 
Linting controllers/index.js ...ERROR
[L173:C24] W117: 'next' is not defined.
        return next(err); 
Linting controllers/index.js ...ERROR
[L257:C28] W117: 'next' is not defined.
          return next(err); 
```

Here is the complete log file:[npm-debug.txt](https://github.com/krakenjs/kraken-example-with-shoppingcart/files/411818/npm-debug.txt)

**Comment by grawk** [comment]
Ah. Good catch. I have always just run the index file directly, and apparently we aren't running a travis build on this repo. Do you have any interest in making a PR to resolve and get added to the contributors in package.json? If not, I'll go ahead and make the change as soon as I can.

**Comment by clayboyd** [comment]
I fixed these issues and a few others in PR #21

**Comment by grawk** [comment]
closed via #21
